packages/core/lottie-react-native.podspec

```diff
@@ -24,7 +29,9 @@ Pod::Spec.new do |s|
 
   # install_modules_dependencies has been defined since React Native 70
   # if you only support React Native 70+, we can remove the `if defined?() else` statement
-  if defined?(install_modules_dependencies)
+  # TODO(workaround): checking RN version for backward comaptibility reasons since lottie is not compatible with RN 72 and below with install_modules_dependencies
+  # TODO: We need to replace this line with   if defined?(install_modules_dependencies)
+  if minor_react_version >= 73
```

Comment:
Maybe check both?
```suggestion
  if minor_react_version >= 73 && defined?(install_modules_dependencies)
```

Reply:
I changed my mind. Do not do the &&. If this method is ever removed, the fallback won't work either, it will silently fail, better to just let it fail loud and have someone open an issue